src/controllers/UsersController.js

```diff
@@ -0,0 +1,74 @@
+import models from '../db/models';
+import TokenAuthenticate from '../helpers/tokenAuthentication';
+import Response from '../helpers/response';
+const { User, Profile } = models;
+import passwordHash from '../helpers/passwordHash';
+import EmailNotificationAPI from '../helpers/EmailNotificationAPI.js'; 
+
+
+/**
+ * @description Defines the actions to for the users endpoints
+ * @class UserController
+ */
+
+
+ class UserController {
+   static async signUp(req, res){
+       let response;
+      const signUpType = 'local';
+    try{
+    const { firstname, lastname, email, username, password, userType, bio, imgURL } = req.body;
+      const hash = await passwordHash(password, 10);
```

Comment:
Please avoid hardcoding values, declare a variable and give them a name. It makes the code easier to read.

Reply:
ok